pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -94,6 +94,12 @@ title: <%= I18n.t('dashboard_landing_title').inspect %>
     };
   });
 
+  filters.filter('getNameById', function() {
+    return function(input, id) {
+        return input.find(function(val) { return val.id == id; }).name;
```

Comment:
I don't believe find is fully supported. Also note that if this is passed an invalid ID, it will error out